src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -1,20 +1,33 @@
-import { pxToRem } from '../../../../lib'
+import { pxToRem, getColorSchemeWithCustomDefaults } from '../../../../lib'
+import { ColorValues, ColorScheme, SiteVariablesPrepared } from '../../../types'
 
-export default () => {
+type LabelColorScheme = Pick<ColorScheme, 'foreground' | 'background'>
+
+export interface LabelVariables {
+  colorScheme: ColorValues<LabelColorScheme>
+  circularRadius: string
+  padding: string
+  startPaddingLeft: string
+  endPaddingRight: string
+  height: string
+  iconColor: string
+}
+
+export default (siteVars: SiteVariablesPrepared): LabelVariables => {
   const color = 'rgba(0, 0, 0, 0.6)'
 
   return {
+    colorScheme: getColorSchemeWithCustomDefaults(siteVars.colorScheme, {
```

Comment:
I am not sure about this idea, can we omit it for initial implementation?

```js
colorScheme: {
  ...siteVars.colorScheme,
  default: {
    ...siteVars.colorScheme,
    foreground: color,
    background: 'rgb(232, 232, 232)',
  }
}
```

Reply:
Not sure then how will the components define their default scheme, as in the label we have grey foreground and background, but for the button is white background and grey foreground...